lib/tilelive.js

```diff
@@ -420,7 +420,7 @@ tilelive.copy = function(src, dst, options, callback) {
         var pipeline = opts.type === 'list' ? opts.listStream.pipe(get) : get;
         if (options.transform) {
             pipeline = pipeline.pipe(options.transform);
-            pipeline.on('error', done);
+            pipeline.on('error', done(err)); // eslint-disable-line no-undef
```

Comment:
I think that what you're trying to do is already being done...

Spelling this out in with more explicit syntax:

```js
pipeline.on('error', function handleError(err) { done(err); });
```

is the same as

```js
function handleError(err) { done(err); }
pipeline.on('error', handleError);
```

or juse

```js
pipeline.on('error', done);
```